tests/e2e/helper/helper_oc.go

```diff
@@ -36,3 +36,21 @@ func OcGetCurrentProject() string {
 	}
 	return ""
 }
+
+// CheckCmdOpInRemoteCmpPod runs the provided command on remote component pod and returns the return value of command output handler function passed to it
+func CheckCmdOpInRemoteCmpPod(cmpName string, appName string, cmd string, checkOp func(cmdOp string, err error) bool) bool {
+	cmpDCName := fmt.Sprintf("%s-%s", cmpName, appName)
+	podName := CmdShouldPass(fmt.Sprintf("oc get pods | grep %s | awk '{print $1}' | tr -d '\n'", cmpName))
```

Comment:
we can't use this. It uses awk, and grep and those won't work on Windows

Reply:
you should be able to use something like this `oc get pods --selector "deploymentconfig=%s" jsonpath='{.metadata.name}'`

Reply:
Ack